simple_ado/__init__.py

```diff
@@ -108,6 +108,56 @@ def verify_access(self) -> bool:
 
         return True
 
+    def create_pull_request(
+        self,
+        source_branch: str,
+        target_branch: str,
+        *,
+        title: Optional[str] = None,
+        description: Optional[str] = None,
+        reviewer_ids: List[str] = [],
```

Comment:
Don't have `[]` as a default parameter. Instead make this optional. 

This explains why: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments